pyclient/docs/source/examples.rst

```diff
@@ -89,11 +80,44 @@ Filter a table
     session = Session()
 
     table = session.empty_table(5)
+
     table = table.update(["Values = i"])
 
     table = table.where(["Values % 2 == 1"])
+
     session.bind_table(name="MyTable", table=table)
 
+Query objects
```

Comment:
this appears twice?

Reply:
Yeah we have both the README and this doc for all of these. They're basically duplicated